AutoRest/Generators/CSharp/CSharp/ClientModelExtensions.cs

```diff
@@ -132,6 +132,36 @@ private static bool ShouldValidate(this IType model)
         }
 
         /// <summary>
+        /// Format the documentation of a property properly with the correct getters and setters. Note that this validation will
+        /// only acryonyms and article words.
+        /// </summary>
+        /// <param name="property">The given property documentation to format</param>
+        /// <returns></returns>
+        public static string GetFormattedPropertyDocumentation(PropertyTemplateModel property)
+        {
+            if (string.IsNullOrEmpty(property.Documentation))
+            {
+                return property.Documentation.EscapeXmlComment();
+            }
+
+            string documentation = property.IsReadOnly ? "Gets " : "Gets or sets ";
+
+            string firstWord = property.Documentation.Split(' ').First();
```

Comment:
Should you trim the start of the Documetation to ensure there aren't any leading spaces?